medicines/api/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM rust:1.40 AS build
+
+WORKDIR /src
+
+RUN cargo build --release
+
```

Comment:
Where'd this `Dockerfile` come from? Should it be part of this PR?

Reply:
Sorry, it came from my previous commit, I removed!